bin/crew

```diff
@@ -1403,12 +1350,12 @@ def install_files(src, dst = File.join( CREW_PREFIX, src.delete_prefix('./usr/lo
         # Check for ACLs support.
         rsync_version = `rsync --version`.chomp
         if rsync_version.include?('ACLs') && !rsync_version.include?('no ACLs')
-          system 'rsync', "-ah#{@verbose}HAXW", '--remove-source-files', src, dst, exception: true
+          system 'rsync', "#{@verbose} -ahHAXW", '--remove-source-files', src, dst, exception: true
```

Comment:
This might not work as `"#{@verbose} -ahHAXW"` is treated as a single parameter, equivalent of `rsync "-v -ahHAXW" --remove-source...` in shell. The `@verbose` needs to be a standalone argument passed to `system`

Reply:
Having rsync "-v -ahHAXW" --remove-source... is the intent, though

Reply:
Yes-- I have removed the version of verbose without the leading `-`, and so I am making these changes to the previous usecases which relied on not having a leading `-`.